Automattic/jetpack
# Get Apps card: Ensure card is always displayed, regardless of promos or dismissal

Following the changes to the Get Apps card in https://github.com/Automattic/jetpack/pull/26276, there is no longer a button to dismiss the card. However, some of the surrounding code related to that preference was not deleted.

In addition, the card is displayed when [the `arePromotionsActive` function](https://github.com/Automattic/jetpack/blob/8b32bb737e90923a229a0f21cf54b7d654b59d24/projects/plugins/jetpack/_inc/client/state/initial-state/reducer.js#L310-L319) is set to false. `arePromotionsActive` returns false in cases where a user can't purchase plans. This isn't necessary, as the apps can be used by all types of users and whether promotions are active isn't relevant to its use. 

#### Changes proposed in this Pull Request:
* All the code related to storing a user's dismissal preference is removed with this PR. This includes `appsCardDismissed()` and `dismiss_dash_app_card` mentions as well as now-unused CSS related to the dismiss button. This will prevent future confusion.
* In addition, the `arePromotionsActive` conditional has been removed from the Get Apps component, so that the card is always displayed.

#### Other information:


#### Jetpack product discussion

N/A

#### Does this pull request change what data or activity we track or use?

N/A

#### Testing instructions:

* Navigate to _Jetpack_ → _Dashboard_ to view the card.
* Locally, set [the `arePromotionsActive` function](https://github.com/Automattic/jetpack/blob/8b32bb737e90923a229a0f21cf54b7d654b59d24/projects/plugins/jetpack/_inc/client/state/initial-state/reducer.js#L310-L319) to return `false`.
* Verify that the banner still displayed. 

#### Screenshot:

<img width="1163" alt="Screenshot 2022-10-06 at 11 57 52" src="https://user-images.githubusercontent.com/2998162/194296165-b67b9ced-50ab-4126-9382-195588dec2ae.png">
